src/_pytest/legacypath.py

```diff
@@ -452,6 +436,29 @@ def pytest_configure(config: pytest.Config) -> None:
     mp.setattr(Node, "fspath", property(Node_fspath, Node_fspath_set), raising=False)
 
 
+@pytest.hookimpl
+def pytest_configure(config: pytest.Config) -> None:
+    """Installs the LegacyTmpdirPlugin if the ``tmpdir`` plugin is also installed."""
+    if config.pluginmanager.has_plugin("tmpdir"):
```

Comment:
The `LegacyTmpdirPlugin` should be installed during `pytest_configure` as that's called after `tmpdir` is installed.